docs/en/packages/fast-application-utilities/README.md

```diff
@@ -0,0 +1,31 @@
+---
+id: index
```

Comment:
I think we should place these in temp directories ignored in the gitignore, it seems like otherwise we will have a lot of unnecessary duplication that needs to be updated.

Reply:
Docusaurus requires they be in this folder in order to compile and auto-generate the navigation menu.

What you're saying makes sense in our long term approach when we remove the dependency on Docsuarus and are in complete control.

Reply:
I'm not saying delete the files, we should just add them to the gitignore since they are autogenerated. I wouldn't think docusaurus cares if they files are being tracked, only if they exist.

Reply:
We can leave them where they are (aka not moved to a temp directory), but have them ignored. I think the globbing pattern would be straightforward docs/en/packages/*/README.md

Reply:
Updated per your request.